projects/packages/forms/tools/webpack.config.contact-form.js

```diff
@@ -135,4 +145,15 @@ module.exports = [
 			} ),
 		],
 	},
+	{
```

Comment:
Build JS files from `src/contact-form/js` and output them in `dist/contact-form/js`.